integration/index_large_cardinality_high_concurrency_test.go

```diff
@@ -0,0 +1,112 @@
+// +build integration_disabled
```

Comment:
Do we want to have this disabled by default?

Reply:
Yep it's too heavy to run on the ci atm. Leaving it in as more a benchmarking utility for now.